### ISSUE TITLE
Setup ISOOKO Repo

### ISSUE THREAD

**Issue by Shadrock** [ushahidi/platform#3216]
**What's needed?**
A github repository for ISOOKO. Use the [repository/respositories for COMRADES as an example](https://github.com/ushahidi?utf8=%E2%9C%93&q=comrades&type=&language=). 

**Due Date**
Thursday, August 23nd, EOD/EST.

**Additional context**
This will be the repository where the code for the "ISOOKO" platform lives and will have an accompanying deployment similar to [https://comrades-stg.ushahidi.com/](https://comrades-stg.ushahidi.com).

**Comment by jrtricafort** [comment]
@rowasc If it's not realistic to get done by tomorrow, that's ok, we just need to come up with a new delivery date and propose to Shadrock.

**Comment by Shadrock** [comment]
Thanks @jrtricafort, sorry about the short-notice on this one. To answer your question, the repo does it need to be populated with all of the boilerplate files/code, along with a new deployment. If the date proposed is unrealistic, please propose a new one and I can message back to the partners.

**Comment by rowasc** [comment]
Thanks. @Shadrock  . Discussing this with Will to come back with a date ASAP

**Comment by rowasc** [comment]
@jrtricafort @Shadrock I think we're still fine for tomorrow, the server is up and running, we just need to adjust some bits for the exporter to work correctly.
I'd like to set thursday EOD as a deadline so we have a little more space to fix any issues that may come up, if that's possible.

**Comment by jrtricafort** [comment]
Thanks @rowasc . @Shadrock let us know if that works for you-- Thursday EOD as an external deadline. We will continue to work on this tomorrow (it is roughly a 1-, 1.5-day effort)

**Comment by rowasc** [comment]
@Eve-wanderi this is ready to test in https://isooko-stg.ushahidi.com/ 

Could you run a smoke test with this testpad script? https://ushahidi.ontestpad.com/script/30
Please note that there’s a section marked to ignore (around line 84 to the end) in this testrun. Those steps (84 to the end of the script) are only used when we want to test deployment creation so we don't need them in this particular case.

**Comment by rowasc** [comment]
@Shadrock this is ready in https://isooko-stg.ushahidi.com/ , the forks are https://github.com/ushahidi/isooko-platform-api and https://github.com/ushahidi/isooko-platform-client. 

Eve and I did a test run (smoke testing only), the details are here if you're interested https://ushahidi.ontestpad.com/script/30#// . 

I found a weird behavior that I think is a bug with category deletion when deleting a parent category with children BUT it's not related to the isooko setup at all so this should be considered done. I'll report the bug separately.
